src/components/UI/Card.tsx

```diff
@@ -1,8 +1,11 @@
 import "./Card.css";
-
-function Card(props) {
+type classProps = {
+  className: string;
+  children: JSX.Element[];
+};
+const Card: React.FC<classProps> = (props) => {
   const classes = "card " + props.className;
   return <div className={classes}>{props.children}</div>;
```

Comment:
There is no need to assign a variable, just to construct the class name. You can use template literals to construct the string with the text and the value of the variable.